packages/core/src/http-client/http-client.ts

```diff
@@ -83,6 +83,25 @@ export function execute(executeFn: ExecuteHttpRequestFn) {
   };
 }
 
+/**
+ *
+ * @experimental This is an experimental function, which might be removed later.
+ * @param destination - A destination or a destination name and a JWT.
+ * @param requestConfig - Any object representing an HTTP request.
+ */
+
```

Comment:
```suggestion
```
I think this will not be recognized if there is an empty line beween

Reply:
In light of the fact that this is only used in tests for now, maybe rather add this there?